### PR TITLE
feature: added alert

### PR DESCRIPTION
**Pull request by omgaikwad**
Screenshot (24)

Suggestions for code improvements are appreciated.

### CODE REVIEW

**Comment by prathmeshjagtap** [comment]
code looks clean and well organized
Go ahead and merge it.

File: components/alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
+/* Alert */
+
+.alert {
+  width: 100%;
+  box-sizing: border-box;
+  font-size: 1rem;
+  background-color: var(--primary-color);
+  color: white;
```
Comment (aryan-13) [inline components/alert/alert.css]: Using variables is a better pratice!

File: components/alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+}
+
+.alert-success {
+  background-color: var(--success-color);
+  color: white;
+}
+
+.alert-warning {
+  background-color: var(--warning-color);
+  color: white;
```
Comment (aryan-13) [inline components/alert/alert.css]: here also, You can define a variable for the white color

File: components/alert/alert.html [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+            <div class="alert alert-danger">This is Danger Alert!</div>
+          </div>
+        </div>
+
+        <div class="code-example-container">
+          <iframe
+            src="https://carbon.now.sh/embed?bg=rgba%28211%2C211%2C211%2C1%29&t=vscode&wt=none&l=auto&width=872&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=0px&ph=15px&ln=false&fl=1&fm=Hack&fs=14.5px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522alert%2520alert-primary%2522%253EThis%2520is%2520Primary%2520Alert%21%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522alert%2520alert-secondary%2522%253EThis%2520is%2520Secondary%2520Alert%21%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522alert%2520alert-success%2522%253EThis%2520is%2520Success%2520Alert%21%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522alert%2520alert-warning%2522%253EThis%2520is%2520Warning%2520Alert%21%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522alert%2520alert-danger%2522%253EThis%2520is%2520Danger%2520Alert%21%253C%252Fdiv%253E"
+            style="
+              width: 872px;
+              height: 154px;
```
Comment (aryan-13) [inline components/alert/alert.html]: Inline styles should be avoided, you can select this iframe in your CSS and then give it the desired width.
Also use of pixels makes it less responsive!

**Comment by t007rushi** [comment]
Looks Good Merge it,One suggestion is to host the site on netlify for ease in demo instead of SS

File: components/alert/alert.css (hunk 1 above)
Comment (omgaikwad) [reply components/alert/alert.css]: Sure, I will make neccessary changes.

File: components/alert/alert.css (hunk 2 above)
Comment (omgaikwad) [reply components/alert/alert.css]: Yes, I'll do the changes.

File: components/alert/alert.html (hunk 3 above)
Comment (omgaikwad) [reply components/alert/alert.html]: Thank you for your suggestions, I will make necessary changes before merging it.

**Comment by omgaikwad** [comment]
> Looks Good Merge it,One suggestion is to host the site on netlify for ease in demo instead of SS

Sure, Thanks for valuable suggestion.

**Review by junaidshaikh-js — COMMENTED** [review]
The code looks good.

**Pull request merged**